packages/zoe/src/zoeService/zoe.js

```diff
@@ -209,6 +209,34 @@ function makeZoe(vatAdminSvc) {
                 brandToPurse.init(brand, E(issuer).makeEmptyPurse());
               }
             }),
+        makeEmptySeat: (keyword, initialAllocation) => {
```

Comment:
If it is an empty seat, why does it have an initial allocation?

Reply:
dropped